.travis.yml

```diff
@@ -33,6 +35,9 @@ services:
   - mysql
   - memcached
 
+before_install:
+  - travis/setup-mysql8.sh $MYSQL8
```

Comment:
Script execution should be added to setup-phpbb.sh:
https://github.com/phpbb/phpbb/blob/master/travis/setup-phpbb.sh

Reply:
MySQL got upgraded on `before_install` part so `setup-phpbb.sh` is already running on MySQL 8, or isn't it?

Reply:
Oh, I guess I see, you mean move it from there to `setup-phpbb.sh`. ok.